EScavange.py

```diff
@@ -0,0 +1,18 @@
+import database
```

Comment:
I believe conventions dictate that this file should be named escavenge.py (all lower-case). Also, make sure you're spelling escavEnge correctly ;P

Reply:
:cry: